install/framework/framework.wxs

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
 	<Product Id="*" 
-             Name="NUnit Framework !(bind.FileVersion.nunit.framework.45.dll)" 
+             Name="NUnit Framework $(var.DisplayVersion)" 
```

Comment:
If that is the case, then how will users know if the have the beta or the final release installed by looking in Add/Remove Programs? Look at the version column?